src/components/layout/Landing.js

```diff
@@ -35,38 +37,94 @@ const Landing = ({ isAuthenticated, quiz, setAgreement }) => {
         as you decide what type of compensation is best for you.
       </Typography>
       <Typography variant="subtitle1" gutterBottom={true}>
-      Disclaimer: This is an educational and informational tool and the information contained within it does in no way constitute legal advice. Any person who intends to use the information contained herein in any way is solely responsible for independently verifying the information and obtaining independent legal or other expert advice if necessary
+        Disclaimer: This is an educational and informational tool and the
+        information contained within it does in no way constitute legal advice.
+        Any person who intends to use the information contained herein in any
+        way is solely responsible for independently verifying the information
+        and obtaining independent legal or other expert advice if necessary
       </Typography>
       <Link to="/terms-and-conditions">Full Terms & Conditions</Link>
       <FormGroup row>
         <FormControlLabel
           control={
-            <Checkbox color="default" checked={quiz.agreement} onClick={onAgreementClick} value="agreement" />
+            <Checkbox
+              color="default"
+              checked={quiz.agreement}
+              onClick={onAgreementClick}
+              value="agreement"
+            />
           }
           label="By checking this box, you acknowledge that you have read and agree to the terms and conditions provided."
         />
       </FormGroup>
       <Button
-      color="primary"
-      component={Link}
-      to={'/quiz'}
-      disabled={!quiz.agreement}
-    >
-      Start
-    </Button>
+        color="primary"
+        variant="contained"
+        component={Link}
+        to={"/quiz"}
+        disabled={!quiz.agreement || quiz.cookies === null}
+      >
+        Start
+      </Button>
+      {quiz.cookies === null && (
```

Comment:
This section feels large enough to be worth extracting to its own component and file.

Reply:
This is a great suggestion, @jamesarosen  - its moved out.